pkg/cluster/gvik.go

```diff
@@ -0,0 +1,37 @@
+package cluster
```

Comment:
Is the name 'gvik.go' intended?

Reply:
sorry, did not push file name fix yesterday.